examples/test-kube-functions-minikube.yaml

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+extra:
+  functionsAsPods: yes
+
+## deployed withh emptyDir
+volumes:
+  persistence: false
+
+# disabled AntiAffinity
+affinity:
+  anti_affinity: false
+
+# disable auto recovery
+components:
+  autorecovery: false
+  functions: yes
+
+zookeeper:
+  replicaCount: 1
+
+bookkeeper:
+  replicaCount: 1
+
+broker:
+  replicaCount: 1
+  configData:
+    ## Enable `autoSkipNonRecoverableData` since bookkeeper is running
+    ## without persistence
+    autoSkipNonRecoverableData: "true"
+    # storage settings
+    managedLedgerDefaultEnsembleSize: "1"
+    managedLedgerDefaultWriteQuorum: "1"
+    managedLedgerDefaultAckQuorum: "1"
+
+proxy:
+  replicaCount: 1
+
+functions:
+  replicaCount: 1
+  runtime: kubernetes
+
+image:
+  zookeeper:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  bookie:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  autorecovery:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  broker:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  functions:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
+  proxy:
+    repository: apachepulsar/pulsar-all
+    tag: 2.8.0-SNAPSHOT
```

Comment:
Is this file intentionally part of this PR?

Reply:
I committed unintentionally other files on my work directory.
Removed